stable/xray/templates/xray-server-statefulset.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         component: {{ .Values.server.name }}
       annotations:
         checksum/config: {{ include (print .Template.BasePath "/xray-setup-conf.yaml") . | sha256sum }}
+      {{- if .Values.velero.restic.enabled}}
```

Comment:
I think this doesn’t make sense. We can’t add support with yaml flags to every annotation option k8s has to offer because it will never end. There are dozens of different annotations you can use to control stuff with k8s and we can’t support every single one with a dedicated flag. This is why we provide the generic option to provide any annotation you want. This is what you should use. I can see that such a feature is missing in the X-ray statefulset, so if you want to contribute a PR, this is what I would do. You can look at the ingress template we have as an example - we provide the option for the user to add any annotation they see fit

Reply:
Probably a better addition! I will look into adding a more generic update!

Reply:
Cool @evannorstrand, thanks for your contribution!